test/test_datasets_video_utils_opt.py

```diff
@@ -1,7 +1,6 @@
 import unittest
 
 import test_datasets_video_utils
-from torchvision import set_video_backend
```

Comment:
I think commenting it would be better, because this will be needed if we remove the "temporary" issue below.